driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java

```diff
@@ -245,6 +253,9 @@ public String asCQLQuery() {
 
     protected StringBuilder appendOptions(StringBuilder sb, boolean formatted) {
         // Options
+        if(options==null){
```

Comment:
In several places the formatting rules are not respected, please run a global format on this PR before merging, applying the rules for driver 3.x [here](https://github.com/datastax/java-driver/blob/3.x/CONTRIBUTING.md#editor-configuration).